.github/workflows/ci.yaml

```diff
@@ -769,32 +698,50 @@ jobs:
         run: "kubectl create namespace demo-targets"
 
       # ---- Install Test Dependencies ----
-
       - name: "Install Test Dependencies"
         run: |
           cd tests/integration/
           npm ci
 
+        # ----- Build Test-Scan ScanType Image ----
+
+      - name: Build Image
+        working-directory: ./parser-sdk/nodejs/
+        run: make docker-build-sdk
+
+      - name: Export Docker Images
+        working-directory: ./parser-sdk/nodejs/
+        run: make docker-export
+
+      - name: Kind import
+        working-directory: ./parser-sdk/nodejs/
+        run: make kind-import
```

Comment:
I think that these two steps are unnecessary - the SDK image does not need to be present in the cluster, it is only needed as a base image to build the parsers later. For the parsers, these two steps are correct, but here they don't hurt, but also don't do anything productive, afaik.

Reply:
You are correct. The steps are now gone.